fitsio/fitslib.py

```diff
@@ -4487,7 +4488,8 @@ def _record2card(self, record):
         v_isstring=isstring(value)
 
         if name == 'COMMENT':
-            card = 'COMMENT   %s' % value
+            #card = 'COMMENT   %s' % value
+            card = 'COMMENT %s' % value
```

Comment:
Did this need two spaces between `COMMENT` and `%s`?

Reply:
the old way added extra spaces; this may have to do with the new way that things are being read

Reply:
Do we need to change the code block with CONTINUE below as well then?

Reply:
I don't know.  That code path is not hit by the tests, I do not think because the CONTINUE are no longer in the list returned from the C code

Reply:
note the only place record2card is used is when you print the header, and that is not intended for use in generating a valid header.

Of course it would be good to get it right

Reply:
OK, just ran a test that shows continues are being printed correctly there